GameLiftPlugin/Scripts~/PACKAGE_FOR_RELEASE.md

```diff
@@ -0,0 +1,77 @@
+# GameLiftServerSDK Unreal Plugin
+## Documention
+You can find the official Amazon GameLift documentation [here](https://aws.amazon.com/documentation/gamelift/).
+
+
+## Using the plugin
+### Building the C++ SDK
+Before we can use the plugin inside an Unreal game, we need to compile the Amazon GameLift server C++ SDK.  
```

Comment:
- In the next line the instructions say "To **build** the ...", in this line it says "need to **compile** the ..."
My suggestion is to change this to "need to **build** the ...".
- Instead of "We", I would use "You" like this >> "Before you can use the plugin inside an Unreal game, you need to build the Amazon GameLift server C++ SDK."

Reply:
changed as suggested